front/src/applications/operationalStudies/components/SimulationResults/ChartHelpers/defineChart.ts

```diff
@@ -19,7 +19,7 @@ const defineChart = (
   rotate: boolean,
   keyValues: string[],
   id: string
-) => {
+): Chart | SpeedSpaceChart => {
```

Comment:
I think SpeedSpaceChart is not very useful here, but it's a detail :) 
```suggestion
): Chart => {
```

Reply:
done :)